src/storage/pubsub/listener.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "index.h"
+
+#include <stream/cc_sockio.h>
+
+/*
+ * a listener is a client that has subscribed to at least one channel
```

Comment:
I thought that a listener may be subscribed to 0 channels (after unsubscribe it is not deleted)?